.markdownlint.json

```diff
@@ -0,0 +1,36 @@
+{
+  "default": true,
+  "MD001": false,
```

Comment:
I wonder if it makes sense to use yaml and add comments to explain what the rules do.

Reply:
I don't think YAML is supported in the Node version, but they do have "aliases" for the rules that are more readable. I just stuck with the rule names for the initial pass to disable the failing ones, but if you want to do a styleguide, the aliases are better